setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
```

Comment:
It's nitpicking, but adding the setup file is not inline with the rest of your changes. Could we do this in a different PR?